crates/ruff/src/rules/pycodestyle/rules/invalid_escape_sequence.rs

```diff
@@ -114,7 +114,10 @@ pub fn invalid_escape_sequence(
                     Range::new(location, end_location),
                 );
                 if autofix {
-                    diagnostic.amend(Fix::insertion(r"\".to_string(), location));
+                    diagnostic.amend(Fix::insertion(
+                        r"\".to_string(),
+                        Location::new(location.row(), location.column() + 1),
```

Comment:
It might _also_ be possible to fix this by giving one rule precedence in `crates/ruff/src/autofix/mod.rs#cmp_fix`, so that we apply the fixes in a specific order even when they're anchored to the same code point. But, what you have here is good too, arguably better.

Reply:
Good callout! `cmp_fix` wasn't there when I forked the repo, I'll make a change there as well. Think I'll still keep my "insert at column 1" change, might help with other rules clashing in a similar way.

I was thinking of how to make the overlap logic more robust, but didn't come up with anything yet. Need to get the hang of the codebase a bit more I think :)